docs/DeveloperGuide.md

```diff
@@ -426,6 +426,12 @@ ShortcutSettings implements the `Serializable` interface, thus is saved to `json
     * Pros: Will use less memory (No extra data structure created).
     * Cons: Difficult to manage duplicate shortcut mappings.
 
+--------------------------------------------------------------------------------------------------------------------
+
+## **Planned Enhancements**
+
+1. DoConnek Pro currently does not check for duplicate persons. This means that people with duplicate parameters (`Name`, `Phone`, `Email` etc.) can be added. 
```

Comment:
I thought it does not allow `Name` as a duplicate parameter?

Reply:
I think we had decided to allow duplicate names because it is possible that people have the same name.
I'm not sure if this PR removed the check for names when adding people?

Reply:
Thanks for checking, will remove the check as intended!